hicommonwealth/commonwealth
# #6511 : Update Editor Component UI - Toolbar Links, Code, and Blockquotes

This PR updates the toolbar icons for creating blockquotes, links, and codeblocks in the editor.

Note that the editor is still being worked on, so will still have elements that are updated in other cards. 

## Link to Issue
Closes: #6511

## Description of Changes
- updates the icons used for blockquotes, links, and codeblocks

## Test Plan
- go to the create thread page
- confirm expected behavior and style for the aforementioned icons
